src/polyfem/solver/forms/adjoint_forms/TransientForm.cpp

```diff
@@ -181,7 +181,7 @@ namespace polyfem::solver
 	void TransientForm::solution_changed(const Eigen::VectorXd &new_x)
 	{
 		AdjointForm::solution_changed(new_x);
-		obj_->solution_changed(new_x);
+		// obj_->solution_changed(new_x);
```

Comment:
need discussion here, many static forms don't need solution_changed_step()

Reply:
I need it for the MeshTargetForm to cache per timestep quantities.